opm/io/eclipse/OutputStream.hpp

```diff
@@ -140,6 +142,17 @@ namespace Opm { namespace ecl { namespace OutputStream {
         void write(const std::string&              kw,
                    const std::vector<std::string>& data);
 
+        /// Write padded character data (8 characters per string)
+        /// to underlying output stream.
+        ///
+        /// Must not be called prior to \c prepareStep
```

Comment:
I though the `prepareStep()` method had been removed / made private?

Reply:
> I though the `prepareStep()` method had been removed

You're right.  I've removed the documentation reference.